cms/views.py

```diff
@@ -348,3 +353,45 @@ def change_password(request):
     context['form'] = form
     context.update(csrf(request))
     return render(request, 'cms/templates/change_password.html', context)
+    
+def confirm_student(request, mdlid):
```

Comment:
Have we handled both student and organiser (basically all type of user's who needs to verify their email id?